src/doodle/canvas.js

```diff
@@ -59,6 +59,7 @@ const Canvas = ({
       onMouseMove={handleMouseMove}
       onMouseUp={handleMouseUp}
       onMouseLeave={handleMouseLeave}
+      id="canvas"
```

Comment:
We have multiple instances of `canvas`, so could introduce some bugs. I'd recommend adding an optional prop to canvas that is `className`, and passing the size class in directly from `DoodleCanvas`.